$OpenBSD: patch-src_runtime_gc-common_c,v 1.4 2021/05/28 16:23:31 solene Exp $
ffsl is non-standard, but both gcc and clang have it as builtin...
clang only has it as builtin

Index: src/runtime/gc-common.c
--- src/runtime/gc-common.c.orig
+++ src/runtime/gc-common.c
@@ -58,6 +58,8 @@
 #define LONG_FLOAT_SIZE 3
 #endif
 
+#define ffsl	__builtin_ffsl
+
 os_vm_size_t dynamic_space_size = DEFAULT_DYNAMIC_SPACE_SIZE;
 os_vm_size_t thread_control_stack_size = DEFAULT_CONTROL_STACK_SIZE;
 
